core/src/main/scala-3/spire/syntax/macros/cforMacros.scala

```diff
@@ -0,0 +1,167 @@
+
+package spire.syntax.macros
+
+import quoted._
+import collection.immutable.NumericRange
+
+import spire.syntax.cfor.{RangeLike, RangeElem}
+
+inline def cforInline[R](init: R, test: R => Boolean, next: R => R, body: R => Unit): Unit =
+  var index = init
+  while (test(index))
+    body(index)
+    index = next(index)
```

Comment:
Do you mean the params? I'm not sure either. This is the first time I do macros and my main guidance has been to make the tests pass.
If making them inline is better let's do that but I have no insights one way or another

Reply:
> main guidance has been to make the tests pass

That's fair, in which case we don't even really need any inline 😉 ok, let's not block on this. It can be a future issue to figure out how to improve this macro. Unless @vasilmkd want's to give us the magic diff 😉

Reply:
No, let's do feature and performance work later on after some stability is established.